src/Dropdown/useClosable.ts

```diff
@@ -0,0 +1,46 @@
+import { useState, useEffect } from 'react'
+
+type returnValue = {
+  isOpen: boolean,
+  open: () => void,
+  close: () => void,
+  toggle: () => void
+}
+
+export default function (
```

Comment:
By convention all UI modules use named exports (so all module members can easily be exported using `export * from './module`